### PR TITLE
add "#include <studio.h>" matcher

### PR DESCRIPTION
**Pull request by brianyu28**
Addressing #42

### CODE REVIEW

File: helpers/clang.py [hunk 1]
```diff
@@ -192,6 +192,22 @@ def help(lines):
...
+    
+    # $ clang foo.c
+    # /tmp/foo-1ce1b9.o: In function `main':
+    # foo.c:1:10: fatal error: 'studio.h' file not found
+    # #include <studio.h>
+    #          ^
+    matches = re.search(r"^([^:]+):(\d+):\d+: fatal error: '(.*)' file not found", lines[0])
+    if matches:
+        after = [
+            "Looks like you're trying to access a file (`{}`) on line {} of `{}` which does not exist.".format(matches.group(3), matches.group(2), matches.group(1))
```
Comment (dmalan) [inline helpers/clang.py]: "What's access mean?" Maybe just say `#include`?

Comment (brianyu28) [inline helpers/clang.py]: Updated the wording.

**Comment by dmalan** [comment]
PS @brianyu28, @dlloyd09, https://twitter.com/katzenmuenster/status/645724776439390208.

**Comment by dlloyd09** [comment]
Ha!

**Pull request merged**